cloudformation/core/functions.yaml

```diff
@@ -291,14 +299,14 @@ Resources:
     Type: AWS::Lambda::Function
     Properties:
       FunctionName: !Sub '${ResourcePrefix}-api'
-      Runtime: python3.12
+      Runtime: !Sub 'python${LambdaPythonVersion}'
       Handler: api.lambda_handler
-      Code: lambda-packages/cirrus-lambda-dist-py3.12-arm64.zip
+      Code: !Sub 'lambda-packages/cirrus-lambda-dist_python-${LambdaPythonVersion}_${LambdaArch}.zip'
```

Comment:
We can't use `!Sub` (or `!Ref`) here if we want to the `package` command to find the local zip file and upload it to S3 for us. We need to either move back to a fixed file name, or manually upload the zip file to S3 and then specify an `S3Bucket` and `S3Key` here.

I'd prefer for `package` to continue to upload our zip.